OMCompiler/SimulationRuntime/c/util/omc_error.c

```diff
@@ -166,6 +168,61 @@ void initDumpSystem()
   useStream[LOG_SUCCESS] = 1;
 }
 
+/* Deactivates streams for logging except for stdout, assert and success. */
+void deactivateLogging() {
```

Comment:
The coding style doesn't match what's used in the file/runtime.

Reply:
It never does, but maybe now it's better